_episodes/01-introduction.md

```diff
@@ -9,21 +9,21 @@ questions:
 objectives:
 - "Define a relational database."
 - "Explain what SQL is and why to use it."
-- "Identify library skills that translate to using SQL"
+- "Identify library and information skills that relate to using SQL"
 keypoints:
 - "SQL is a powerful language used to interrogate and manipulate relational databases."
-- "People working in library- and information-related roles have skills that allow them to transition into using SQL to organize and access data."
+- "People working in library- and information-related roles have skills that allow them use SQL to organize and access data."
```

Comment:
I think you need to add "to" to "...allow them to use..."?

Reply:
thanks for catching that!